test/test_filters.py

```diff
@@ -136,3 +136,54 @@ def test_b(request, case):
     result = testdir.runpytest("-v", "-s")
     result.assert_outcomes(passed=2)
     result.stdout.re_match_lines([r"Hypothesis calls: 2$"])
+
+
+def test_operationId_filter_with_openapi(testdir):
+    parameters = {"responses": {"200": {"description": "OK"}}}
+    testdir.make_test(
+        """
+@schema.parametrize(operationId="bar_get")
+@settings(max_examples=1)
+def test_(request, case):
+    request.config.HYPOTHESIS_CASES += 1
+    assert case.path == "/v1/bar"
+    assert case.method == "GET"
+""",
+        paths={
+            "/foo": {"get": {**parameters, "tags": ["foo", "baz"], "operationId": "foo_get"}},
+            "/bar": {"get": {**parameters, "tags": ["bar", "baz"], "operationId": "bar_get"}},
+        },
+        schema_name="simple_openapi.yaml",
+    )
+
+    result = testdir.runpytest("-v", "-s")
+    result.assert_outcomes(passed=1)
+    # Then only tests for this tag should be generated
+    result.stdout.re_match_lines([r"test_operationId_filter_with_openapi.py::test_[GET:/v1/bar] PASSED"])
+
+
+def test_operationId_list_filter_with_openapi(testdir):
+    parameters = {"responses": {"200": {"description": "OK"}}}
+    testdir.make_test(
+        """
+@schema.parametrize(operationId=["foo_get", "foo_post"])
+@settings(max_examples=1)
+def test_(request, case):
+    request.config.HYPOTHESIS_CASES += 1
+    assert case.path == "/v1/foo"
+""",
+        paths={
+            "/foo": {
+                "get": {**parameters, "tags": ["foo", "baz"], "operationId": "foo_get"},
```

Comment:
Tags are redundant here and can be deleted to improve readability

Reply:
Yes, I modified it